packages/server/api/forum/messageApi.ts

```diff
@@ -1,8 +1,59 @@
+import {sequelizeToObject} from '../utils/sequelizeToObject';
 import {Messages, Users, MessagesReactions} from '../models';
 import type {TMessage} from '../models';
 import type {TApiResponseData} from '../typing';
 import type {TMessagesRespond} from './typing';
 
+const messageInfoRequest = (user_id: number) => {
+	return [
+		{
+			model: Users,
+		},
+		{
+			model: MessagesReactions,
+			attributes: [
+				'reaction_id',
+			],
+			required: false,
+		},
+		{
+			model: MessagesReactions,
+			as: 'CurrentUserReaction',
+			required: false,
+			where: {user_id},
+			include: [
+				{
+					model: Users,
+				},
+			],
+			limit: 1,
+		},
+	];
+};
+
+const messageWithCountedReactions = (message: TMessagesRespond[0]) => {
+	if (message.MessagesReactions) {
+		const reactions: number[] = [];
+		message.MessagesReactions?.forEach(
+			reaction => reactions.push(reaction.reaction_id));
```

Comment:
Может быть так, если в одну строку не входит (но это мелочи 😄)?
```suggestion
		message.MessagesReactions?.forEach(reaction => (
			reactions.push(reaction.reaction_id)
		));
```

Reply:
Дело вкуса)